fix: Wipe 3s off startup time with lazy kzg init.

charlielye opened the pull request:
Don't compute the kzg table stuff at global level to improve start times.

ludamad reviewed (approved):
Leaving a hanging approve but I'd like this to be warmed up in prover startup codepaths and sequencer startup codepath if needed

charlielye commented:
* **#18862** <a href="https://app.graphite.com/github/pr/AztecProtocol/aztec-packages/18862?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#18861** <a href="https://app.graphite.com/github/pr/AztecProtocol/aztec-packages/18861?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#18860** <a href="https://app.graphite.com/github/pr/AztecProtocol/aztec-packages/18860?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#18859** <a href="https://app.graphite.com/github/pr/AztecProtocol/aztec-packages/18859?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#18858** <a href="https://app.graphite.com/github/pr/AztecProtocol/aztec-packages/18858?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#18813** <a href="https://app.graphite.com/github/pr/AztecProtocol/aztec-packages/18813?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a> 👈 <a href="https://app.graphite.com/github/pr/AztecProtocol/aztec-packages/18813?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `next`


<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.
<!-- Current dependencies on/for this PR: -->

charlielye committed:
fix: Wipe 3s off startup time with lazy kzg init.

Please read [contributing guidelines](CONTRIBUTING.md) and remove this line.

For audit-related pull requests, please use the [audit PR template](?expand=1&template=audit.md).
